production-deployment-using-tiup.md

```diff
@@ -83,9 +83,21 @@ Perform the following steps in this section to deploy a TiDB cluster offline usi
 
 #### Prepare the TiUP offline component package
 
-Method 1: On the [official download page](https://www.pingcap.com/download/), select the offline mirror package (TiUP offline package included) of the target TiDB version. Note that you need to download the server package and toolkit package at the same time.
+**Method 1**: Download the offline mirror packages (TiUP offline package included) of the target TiDB version using the following links. You need to download the server package and toolkit package at the same time. Note that your downloading means you agree with the [Privacy Policy](https://www.pingcap.com/privacy-policy/).
 
-Method 2: Manually pack an offline component package using `tiup mirror clone`. The detailed steps are as follows:
+```
+https://download.pingcap.org/tidb-community-server-{version}-linux-{arch}.tar.gz
+```
+
+```
+https://download.pingcap.org/tidb-community-toolkit-{version}-linux-{arch}.tar.gz
+```
+
+> **Tip:**
+>
+> `{version}` in the link indicates the version number of TiDB and `{arch}` indicates the architecture of the system, which can be `amd64` or `arm64`. For example, the download link for `v7.5.0` in the `amd64` architecture is `https://download.pingcap.org/tidb-community-toolkit-v7.5.0-linux-amd64.tar.gz`.
```

Comment:
Do we need to use `v8.0.0` for the docs master branch?

Reply:
Either `v8.0.0` or `v7.5.0` is ok. But for versions earlier than v7.5, I'll update this version number in cherry picked PRs.